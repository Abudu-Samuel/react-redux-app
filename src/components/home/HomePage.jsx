import React, { Component } from 'react';
import { Link } from 'react-router-dom';

class HomePage extends Component {
  render() {
    return (
      <div className="jumbotron">
        <h1>Pluralight Adminstratrion</h1>
        <p>React, Redux and React router dom in es6</p>
        <Link to="about" className="btn btn-primary btn-lg">Learn More</Link>
      </div>
    );
  }
}

export default HomePage;
