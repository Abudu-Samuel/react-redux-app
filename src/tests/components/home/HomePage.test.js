/* eslint-disable */
import HomePage from '../../../components/home/HomePage';

describe('<HomePage />', () => {
  it('render HomePage component correctly', () => {
    const wrap = shallow(<HomePage />);
    expect(toJson(wrap)).toMatchSnapshot();
  });
});