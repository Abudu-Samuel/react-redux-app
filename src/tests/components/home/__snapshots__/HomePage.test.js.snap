// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<HomePage /> render HomePage component correctly 1`] = `
<div
  className="jumbotron"
>
  <h1>
    Pluralight Adminstratrion
  </h1>
  <p>
    React, Redux and React router dom in es6
  </p>
  <Link
    className="btn btn-primary btn-lg"
    replace={false}
    to="about"
  >
    Learn More
  </Link>
</div>
`;
